filament/src/driver/ExternalContext.cpp

```diff
@@ -72,7 +74,9 @@ ExternalContext* ExternalContext::create(Backend* backend) noexcept {
             return nullptr;
         #endif
     }
-    #if defined(ANDROID)
+    #if defined(USE_EXTERNAL_GLES3)
+        return nullptr;
```

Comment:
Why can't we provide a context manager for this case?

Reply:
The custom ExternalContext I'm using is tied to main(), in the sense that it needs to know enough about the fake-window pointer passed into `createSwapChain` so it can figure out what kind of surface to make.